chart/templates/secrets/redis-secrets.yaml

```
@@ -55,7 +55,7 @@ data:
   password: {{ (default $random_redis_password .Values.redis.password) | b64enc | quote }}
 ---
 {{- end }}
-{{- if not .Values.data.brokerUrlSecretName }}
+{{- if and (not .Values.data.brokerUrlSecretName) (ne .Values.executor "LocalExecutor") (ne .Values.executor "KubernetesExecutor") }}
```

executor can be list so this will pass for `LocalExecutor,KubernetesExecutor` for example.
I would change it to this
```suggestion
{{- if and (not .Values.data.brokerUrlSecretName) (or (contains "CeleryExecutor" .Values.executor) (contains "CeleryKubernetesExecutor" .Values.executor)) }}
```